sheet2/D.cpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+#include <iomanip>
+#include <cmath>
+using namespace std;
+ 
+int main()
+{
+    
+    long long int a,b,c,d;
+    cin>>a>>b>>c>>d;
+    if( (a+b-c)==d || (a+b*c)==d  ||  (a-b*c)==d  || (a-b+c)==d || (a*b+c)==d || (a*b-c)==d ){
```

Comment:
I like the systematic way in which you wrote the cases. Like listing permutations systematically ;)